roles/frameworks/templates/cassandra-config.j2

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
should we expose the other config vars here or just leave it for now? https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/C/cassandra/0/config.json